app/pods/cases/case/subcases/overview/template.hbs

```diff
@@ -6,11 +6,41 @@
   @onSaveCase={{perform this.saveCase}}
 />
 <div class="auk-scroll-wrapper__body">
-  {{#if this.model}}
-    <Subcases::SubcasesProcess @case={{this.case}} @subcases={{this.model}} />
+  {{#if @model}}
+    <Subcases::SubcasesProcess @case={{this.case}} @subcases={{@model}} />
   {{else}}
     <div class="auk-u-m-8">
       <Auk::EmptyState @message={{t "no-subcases"}} />
     </div>
   {{/if}}
-</div>
\ No newline at end of file
+</div>
+{{#if @model}}
+  <hr/>
+  <div class="auk-u-mx-4">
+    <Auk::Toolbar as |Toolbar|>
+      <Toolbar.Group @position="left" as |Group|>
+        <Group.Item>
+          <Auk::Pagination
+            @page={{this.page}}
+            @size={{this.size}}
+            @nbOfItems={{@model.length}}
+            @total={{@model.meta.count}}
+            @onNextPage={{this.nextPage}}
+            @onPreviousPage={{this.prevPage}}
+          />
+        </Group.Item>
+      </Toolbar.Group>
+      <Toolbar.Group @position="right" as |Group|>
+        <Group.Item>
+          {{t "amount-showed"}}
```

Comment:
forgot about this, "dossiers per pagina" => "procedurestappen per pagina"?

Reply:
thanks! made it "Aantal resultaten per pagina", same thing we use in agendas overview